src/util/duration.h

```diff
@@ -136,19 +157,19 @@ class Duration {
     }
 
     QString formatNanosWithUnit() const {
-        return QString("%1ns").arg(toNanos());
+        return QString("%1ns").arg(toIntegerNanos());
```

Comment:
> This is not a question of a personal preference of one of us.

Whether we obey this NIST standard in our logging is actually a question of preference.